### ISSUE TITLE
Discovery tab

### ISSUE THREAD

**Issue by campbefs** [campbefs/cuisfeed#9]
*Browse page similar to Wayfair category pages.
*Grid format. 
*Categories at the top.  
*Possibly filters on the lefthand side, which would require collapsing the lefthand nav. Capturing additional fields in Recipe tab to filter on
*Sort by functionality

*This might require capturing almost all of Edamame's recipes on the server. Contingent on a separate Issue to capture all 10000 recipes from the API
*Consider having another tab where users can see the most likes people on that platform with some of their posts and the option to follow. 
*Popular sub-Tab. This could be a tab that surfaces popular posts, both posts and recipes but in a diff format. This is where you want it to look like Pinterest as much as possible to capture people's eye.
